packages/lib/CalEventParser.ts

```diff
@@ -14,7 +14,7 @@ const translator = short();
 export const getWhat = (calEvent: CalendarEvent, t: TFunction) => {
   return `
 ${t("what")}:
-${calEvent.type}
```

Comment:
`type` used to be the event title but this got changed here: https://github.com/calcom/cal.com/pull/12290